gast/ast3.py

```diff
@@ -226,6 +237,18 @@ def visit_comprehension(self, node):
             return ast.copy_location(new_node, node)
 
     if 8 <= sys.version_info.minor < 12:
+
+        def visit_ClassDef(self, node):
```

Comment:
if I'm not mistaken, it's not only for `8 <= sys.version_info.minor < 12` but just for `sys.version_info.minor < 12`, right?

otherwise looks good.

Reply:
Yes, the same code now exists in both `8 <= sys.version_info.minor < 12` and `sys.version_info.minor < 8` branches, I'll put them together.